pkg/controller/networkpolicy/crd_utils.go

```diff
@@ -140,8 +141,12 @@ func toAntreaIPBlockForCRD(ipBlock *v1alpha1.IPBlock) (*controlplane.IPBlock, er
 // It is used when peer's Namespaces are not matched by NamespaceMatchTypes, for which the controlplane
```

Comment:
It's not related, but can you fix the comment typo? ` toAntreaPeerForCRD creates an Antrea controlplane `, thanks.

Reply:
Fixed